src/views/CheckoutView.vue

```diff
@@ -47,11 +51,27 @@ export default {
     ...mapState('cart', {
       registeredEvents: 'registeredEvents',
     }),
+    ...mapState('user', {
+      adminLevel: 'adminLevel',
+    }),
   },
   methods: {
     ...mapMutations('cart', {
       cancelRegistration: 'cancelRegistration',
     }),
+    async onClickCheckout() {
+      if (USER[this.adminLevel] === USER[ROLE.ADMIN] || USER[this.adminLevel] === USER[ROLE.PF]) {
+        API.createEventRegistration(this.registeredEvents).then(() => {
```

Comment:
isn't this route protected anyway

Reply:
Yeah, we just want to make sure we're calling the right one for the user